src/Sructs.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
It should be a header file (in include folder), not a C file!
And the file name should not start with a capital charr.
And see if you can find a more describing name, because it contains only the table and friends structs. (I can't think of something)